jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/WebAppContext.java

```diff
@@ -746,13 +740,13 @@ public String[] getServerClasses()
     }
 
     @Override
-    public boolean isServerClass(Class<?> clazz)
+    public boolean isHiddenClass(Class<?> clazz)
     {
         return _serverClasses.match(clazz);
     }
 
     @Override
-    public boolean isSystemClass(Class<?> clazz)
+    public boolean isProtectedClass(Class<?> clazz)
```

Comment:
Still need original isSystemClass(Class<?> clazz) implemented and deprecated.

Reply:
added via default methods on interface